docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -33,9 +33,16 @@ elastic-agent install \
   --unprivileged
 ----
 
-IMPORTANT: On Linux systems, once {agent} has been installed with the `--unprivileged` flag, all {agent} commands that you run should not be prefixed with `sudo`.
+[IMPORTANT] 
+====
+Note the following current restrictions for running {agent} in `unprivileged` mode:
+
+* On Linux systems, once {agent} has been installed with the `--unprivileged` flag, all {agent} commands that you run should not be prefixed with `sudo`.
 Including `sudo` in a command may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
-The `sudo` option is still required for the `elastic-agent install` command.
+* The `sudo` option is still required for the `elastic-agent install` command.
+* Currently, when you run some {agent} commands as non-root, for example, `elastic-agent inspect`, you may need to prefix the command with `sudo -u elastic-agent-user elastic-agent`.
+For example, `sudo -u elastic-agent-user elastic-agent inspect`.
+====
```

Comment:
We can clarify this I think. 

Running as `sudo` by default executes the command as the root user. Running as `sudo -u elastic-agent-user` runs the command as the `elastic-agent-user` that is used to run the agent service when installed with unprivileged. `sudo` is used to execute a command as another user and defaults to root. For an unprivileged agent using sudo is always necessary if the current logged in user is not `elastic-agent-user` or added to the `elastic-agent` group.

I suggest re-wording this to say something like:

* On Linux systems, once {agent} has been installed with the `--unprivileged` flag, all {agent} commands can be run without being the root user. 
  * The `sudo` option is still required for the `elastic-agent install` command. Only root can install new services. The installed service will not run as the root user.
* Using `sudo` without specifying an alternate non-root user with `sudo -u` in a command may result in <<agent-sudo-error,an error>> due to the agent not having the required privileges.
* Using `sudo -u elastic-agent-user` will execute commands as the user running the Elastic Agent service and will always work.
* For files that allow users in the `elastic-agent` group access, using an alternate user that has been added to that group will also work. There are still some commands that are only accessible to the `elastic-agent-user` that runs the service.
  * For example, `elastic-agent inspect`, requires you to prefix the command with `sudo -u elastic-agent-user`. For example, `sudo -u elastic-agent-user elastic-agent inspect`.

Reply:
Thanks @cmacknz. That perfect - so much more clear!

I've updated the screenshot in the description so you can see the final output.

Reply:
No problem, 99% of the problems with unprivileged mode is that it forces users to have a detailed understanding of their operating system's privilege model which is not necessarily easy once you get into the fine details.